### ISSUE TITLE
Alt text for images

### ISSUE THREAD

**Issue by PiotrKozlowski** [openstax-poland/adaptarr-front#149]
Alt text for images should be displayed under the image and editable from this place.

**Comment by MartaSztuka** [comment]
Alt text is here and can be edited.
![Screenshot from 2019-10-08 13-48-03](https://user-images.githubusercontent.com/47241114/66393259-5b8ecd00-e9d2-11e9-9f53-13e953ad50a2.png)
